lib/util.js

```diff
@@ -1153,4 +1153,4 @@ module.exports = exports = {
   puts: deprecate(puts,
                   'util.puts is deprecated. Use console.log instead.',
                   'DEP0027')
-};
+};
```

Comment:
nit: this needs a blank line at the end.

Reply:
Apologies using an editor that formats on save